docs/Guides/QuickStart.md

```diff
@@ -23,6 +23,9 @@ Otherwise, follow these steps:
   - Note: it is recommended that you use sftp to transfer the pax file from a non-z/OS machine to z/OS. This will ensure that there is no ASCII/EBCDIC conversion.
 - On z/OS, expand the pax file: `pax -rf meta-<version>.pax.Z`. 
 - cd to the unpax'ed directory: `cd meta-<version>`
+- Make sure that auto-conversion is enabled by checking the environment variables _BPXK_AUTOCVT and _CEE_RUNOPTS. If not set please run
+  - export _BPXK_AUTOCVT="ON"
+  - export _CEE_RUNOPTS="FILETAG(AUTOCVT,AUTOTAG)"
```

Comment:
Looks good, I would suggest you move above "If you have curl and bash on your system, you can use this one liner" since it could also affect the one-liner.